src/api.v2/helpers/worker/workSubmit/submitMarkerHeatmapWork.js

```diff
@@ -1,6 +1,31 @@
-const getExtraDependencies = require('./getExtraDependencies');
 const submitWork = require('./submitWork');
+const workerVersions = require('../workerVersions');
 
+const MarkerHeatmap = 'MarkerHeatmap';
+
+const getClusteringSettings = (message) => {
+  const { input: { config: { clusteringSettings } } } = message;
```

Comment:
Wouldn't it be cleaner if you just do `const {clusteringSettings} = message.input.config` same thing for the other vars which are destructured like that.

Reply:
I agree -- less nesting and brackets is always cleaner